python/trace-python.c

```diff
@@ -886,33 +858,32 @@ static struct uftrace_python_symbol *convert_function_addr(PyObject *frame, PyOb
 	}
 	else {
 		code = args;
-		Py_INCREF(code);
 	}
 
+	if (is_pyfunc)
+		func_name = get_python_funcname(frame, code, &is_main);
+	else
+		func_name = get_c_funcname(frame, code);
```

Comment:
After this, it should decrease the reference count of `code` as it's not used anymore.

Reply:
added it